packages/bot/src/stores/quick-strategy-store.js

```diff
@@ -98,41 +111,153 @@ export default class QuickStrategyStore {
         this.toggleStrategyModal();
     }
 
+    getSizeDesc = index => {
+        switch (index) {
+            case 0:
+                return localize('The multiplier amount used to increase your stake if you’re losing a trade.');
+            case 1:
+                return localize('The amount that you may add to your stake if you’re losing a trade.');
+            case 2:
+                return localize('The amount that you may add to your stake after each successful trade.');
+            default:
+                return '';
+        }
+    };
+
+    getSizeText = index => {
+        switch (index) {
+            case 0:
+                return localize('Size');
+            case 1:
+            case 2:
+                return localize('Units');
+            default:
+                return '';
+        }
+    };
+
+    validateQuickStrategy = values => {
+        const errors = {};
+        const number_field = ['duration', 'stake', 'size', 'profit', 'loss'];
+    
+        Object.keys(values).forEach(key => {
+            const value = values[key];
+    
+            if (number_field.includes(key)){
+                if (isNaN(value)) {
+                    errors[key] = localize('Must be a number');
+                }
+    
+                if (value <= 0){
+                    errors[key] = localize('Must be a number higher than 0');
+                }
+    
+                if (/^0+(?=\d)/.test(value)) {
+                    errors[key] = localize('Invalid number format');
+                }
+            }
+    
+            if (value === '') {
+                errors[key] = localize('Field cannot be empty');
+            }
```

Comment:
You can use an `else if` construction here for all statements.

Reply:
Sure, make sense to use `else if` since there will be only 1 error message.

Reply:
@sam-binary ?